frontend/apps/reader/modules/readergesture.lua

```diff
@@ -330,6 +339,9 @@ function ReaderGesture:buildMenu(ges, default)
         {"toggle_gsensor", Device:canToggleGSensor()},
         {"toggle_rotation", not self.is_docless, true},
         {"toggle_reflow", not self.is_docless, true},
+        {"toggle_wifi", Device:hasWifiToggle() and "toggle_wifi", true},
+        {"wifi_on", Device:hasWifiToggle() and "wifi_on", true},
+        {"wifi_off", Device:hasWifiToggle() and "wifi_off", true},
```

Comment:
Looks strange. Shouldn't they be `{string action_name, boolean enabled, boolean separator}` ?
And I guess the separator could be a bit moved with the added stuff.